app/models/piece.rb

```diff
@@ -13,6 +14,22 @@ def move_to!(new_row, new_col)
     end
   end
 
+  def capture_en_passant(row_dest, col_dest)
+    @last_updated = Piece.where(game_id: game_id).order("updated_at desc").first
+    return if @last_updated.nil?
+    if Game.find(game_id).black_player_id == user_id 
+       puts "Hi"
+      if row_dest == (@last_updated.row_position - 1) && col_dest == @last_updated.col_position
```

Comment:
Should you not first check if your piece is beside `@last_updated`?  That would let you know if you could capture or not.